src/vs-picgo/index.ts

```diff
@@ -12,191 +12,191 @@ import { formatParam, formatString, showInfo, showError, getUploadedName } from
 
 const _ = require('lodash');
 const _db = require('lodash-id');
+const nls = require('../../package.nls.json');
 _.mixin(_db);
 
 const writeFileP = promisify(fs.writeFile);
 const readFileP = promisify(fs.readFile);
 
 export interface INotice {
-  body: string;
-  text: string;
-  title: string;
+	body: string;
```

Comment:
Why are you using hard tabs? The commit is too verbose, did you format your code with another formatter? I personally sed `Prettier`.

Reply:
I think just a force push after `git commit --amend` is okay if you have fixed the style.

Reply:
Done!

Reply:
Really a meritorious effort! 👍